infra/digichat-release/.env.profile-a.example

```
@@ -51,7 +51,7 @@ DIGISMITH_URL=
 # Chat-only digigraph (defaults match compose.profile-a.yml)
 DIGI_PROJECT_CONFIG=/app/config/digiproject.yaml
 DIGI_WORKFLOW_PROFILE=research_rag
-DIGI_ALLOWED_TOOLS=digisearch,digivault_search_notes
+DIGI_ALLOWED_TOOLS=digisearch,digivault_search_notes,digivault_get_note
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DIGI_ALLOWED_TOOLS key should go before the DIGI_PROJECT_CONFIG key

(UnorderedKey)